easybuild/easyconfigs/j/jhbuild/jhbuild-3.15.92-GCCcore-4.9.3.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'jhbuild'
+version = '3.15.92'
+
+homepage = 'https://wiki.gnome.org/action/show/Projects/Jhbuild'
+description = """JHBuild allows you to automatically download and compile “modules” (i.e. source code packages). 
+Modules are listed in “module set” files, which also include dependency information so that JHBuild can discover 
+what modules need to be built and in what order."""
+
+toolchain = {'name': 'GCCcore', 'version': '4.9.3'}
+
+sources = ['%(version)s.tar.gz']
+source_urls = [('https://github.com/GNOME/jhbuild/archive/')]
+
+# jhbuild is a python script so it has a python dependency.
+# allow use of system Python
+allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
+
+builddependencies = [
+    ('binutils', '2.25')
+]
+
+buildininstalldir=True
```

Comment:
Yeah this is a weird one, the autogen has some dependencies on autotools but they can be ignored (it just leads to more deps). This way is the easiest to do it, it doesn't need anything but you need to build in the install dir because there is some linking to there going on. For details see,
https://developer.gnome.org/jhbuild/stable/getting-started.html.en